server/src/test/java/org/elasticsearch/index/mapper/vectors/SparseVectorFieldTypeTests.java

```diff
@@ -40,4 +40,26 @@ public void testIsNotAggregatable() {
         MappedFieldType fieldType = new SparseVectorFieldMapper.SparseVectorFieldType(indexVersion, "field", false, Collections.emptyMap());
         assertFalse(fieldType.isAggregatable());
     }
+
+    public static SparseVectorFieldMapper.SparseVectorIndexOptions randomSparseVectorIndexOptions() {
+        if (randomBoolean() == false) {
```

Comment:
This is super, super nitpicky but I'd remove the `== false` here just because it really _doesn't_ matter, but helps with readability not to have equality comparisons that don't actually matter.